frameworks/digital-outcomes-and-specialists/questions/briefs/backgroundInformation.yml

```diff
@@ -14,5 +14,5 @@ validations:
     name: answer_required
     message: 'You need to answer this question.'
   -
-    name: under_50_words
+    name: under_100_words
     message: 'Your answer must be no more than 50 words.'
```

Comment:
Aargh, sorry - yes I've changed it back and forth a couple of times but missed it the important time :(